src/output/log.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-console */
+
+import chalk from 'chalk';
+
+const libraryName = '[next-runtime-env]';
+const librarySeperator = '-';
+const libraryPrefix = `${libraryName} ${librarySeperator}`;
+
+const prefixes = {
+  warn: `${chalk.yellow(`warn`)}  - ${libraryPrefix}`,
+  info: `${chalk.cyan(`info`)}  - ${libraryPrefix}`,
```

Comment:
I think there is a double space before the `-`.

Reply:
Yes, that's correct. It aligns the messages in the terminal for better readability 👇🏻

![screenshot](https://user-images.githubusercontent.com/1295837/235600185-e11efdad-d9b2-47fb-b98b-6fc984b167a2.jpg)

Reply:
I see; then we should add a comment, maybe? 😄

Reply:
Commend added 💡